### PR TITLE
[myStrom] Corrected readMe, mystrompir is not a bulb.

### PR DESCRIPTION
**Pull request by Pumpur**
Hi there,
I recently made contribution to the myStrom binding, I just noticed a mistake in the readme file.

Sorry about that.

Signed-off-by: Stefan Navratil <stefan@navratil.ch>

### CODE REVIEW

**Comment by Pumpur** [comment]
Hi @jlaur, I just found a small mistake, which I made in the  #13201 PR, sorry about that.

File: bundles/org.openhab.binding.mystrom/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ This bundle adds the following thing types:
 | ----------------------| ----------- | -------------------------------------------------- |
 | myStrom Smart Plug    | mystromplug | A myStrom smart plug                               |
 | myStrom Bulb          | mystrombulb | A myStrom bulb                                     |
-| myStrom Motion Sensor | mystrompir  | A myStrom bulb                                     |
+| myStrom Motion Sensor | mystrompir  | A myStrom PIR motionsensor                         |
```
Comment (jlaur) [inline bundles/org.openhab.binding.mystrom/README.md:14]:
```suggestion
| myStrom Motion Sensor | mystrompir  | A myStrom PIR motion sensor                        |
```

**Review by jlaur — APPROVED** [review]
Thanks! One minor comment to be addressed.

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks! One minor comment to be addressed.

**Comment by jlaur** [comment]
@Pumpur - no problem, and thanks for fixing it. Sorry for the PR handling, I accidentally clicked the wrong button when reviewing. I committed a small correction to the spelling, and the PR is now merged.

**Review by jlaur — APPROVED** [review]

**Pull request merged**